cinema_service/settings.py

```diff
@@ -125,14 +125,18 @@
 
 USE_I18N = True
 
-USE_TZ = False
+USE_TZ = True
 
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
 STATIC_URL = "static/"
 
+MEDIA_ROOT = BASE_DIR / "media"
```

Comment:
The `MEDIA_ROOT` is set to `BASE_DIR / 'media'`. Ensure that this directory exists and is writable by the server to handle media file uploads correctly.